packages/fcl/src/config-utils.js

```diff
@@ -32,11 +33,15 @@ export async function configLens(regex) {
 }
 
 export async function buildAuthnConfig() {
-  const discoveryWallet = await config.first([
+  const discoveryWalletUrl = await config.first([
     "discovery.wallet",
     "challenge.handshake",
   ])
 
+  const include = await config.get("discovery.authn.include", [])
+  const queryParams = constructApiQueryParams({include})
+  const discoveryWallet = `${discoveryWalletUrl}${queryParams}`
```

Comment:
Will this become `discoveryWallet = discoveryWalletUrl + "?"` if the queryParams are empty?

Reply:
Yeah but won't affect the outcome @JeffreyDoyle

Reply:
Maybe check if any exist and return ''"

Reply:
Ok @JeffreyDoyle @gregsantos updated to return empty string if nothing